### ISSUE TITLE
[Bug]: eks bvt regression: select sql_source_type="cloud_user_sql" and status != "Running" in statement_info results error

### ISSUE THREAD

**Issue by heni02** [matrixorigin/matrixone#9924]
### Is there an existing issue for the same bug?

- [X] #9925

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):nightly-39fde98
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

job：https://github.com/matrixorigin/mo-nightly-regression/actions/runs/5200920046/jobs/9380442078
![image](https://github.com/matrixorigin/matrixone/assets/113406637/3cce389f-20c5-4881-bea9-f5251b456988)
actual  results less than expect results：
![image](https://github.com/matrixorigin/matrixone/assets/113406637/9ab7e1f8-afa3-4940-b7ca-7e6c29292b04)

get cn dn log：
![image](https://github.com/matrixorigin/matrixone/assets/113406637/e5b88c9d-e589-4265-a1be-6d01628d1508)

**Comment by heni02** [comment]
@xzxiong plz confirm

**Comment by xzxiong** [comment]
@gavinyue it look like statement_info has been write into csv

**Comment by gavinyue** [comment]
Will solve this once the logging performance issue is solved.

**Comment by gavinyue** [comment]
Wait for insert optimization and aggregate done.

**Comment by gavinyue** [comment]
Stilling pending on #9834

**Comment by gavinyue** [comment]
make the progress update on https://github.com/matrixorigin/matrixone/issues/9834

**Comment by gavinyue** [comment]
等待PR merge

**Comment by gavinyue** [comment]
https://github.com/matrixorigin/matrixone/pull/10061. Almost ready,

**Comment by gavinyue** [comment]
Still pending

**Comment by gavinyue** [comment]
Pending for the stmt_aggr pr merge. Stuck on dn oom

**Comment by gavinyue** [comment]
stmt_aggr merged into main.  Could you recheck in nightli-regression? @heni02  Thanks.

**Comment by gavinyue** [comment]
Need verification.

**Comment by heni02** [comment]
@gavinyue Is there a simplified case that can be reproduce？

**Comment by gavinyue** [comment]
暂时没重现

**Comment by heni02** [comment]
This error has not occurred recently on eks bvt test. Will there be a simple case recurrence to be supplemented in the future
https://github.com/matrixorigin/matrixone/actions/runs/5419272342/jobs/9852298290
